### PR TITLE
Update dg

### LINKED ISSUE #236: Update PPP and DG for Jason

### PR DESCRIPTION
**Pull request by JasonCP14**
Resolve #236 
* Add sequence diagram for `Import` command
* Add manual testing for `Add` and `Import` command
* Add TOC
* Update PPP

### CODE REVIEW

**Review by maryjess — APPROVED** [review]
Sorry - missed something, now requesting changes

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -341,43 +322,31 @@ Given below is an example success scenario and how the `export` mechanism behave
 7. The displayedList is stored as a JSON file in `data/export/<currentDateTime>.json`.
 8. A `CommandResult` object indicating that the `export` command is successful will be created.
 
-The following sequence diagram shows how the `export` command works:
```
Comment (maryjess) [inline docs/DeveloperGuide.md:344]: Should this be `export` instead?

**Review by maryjess — COMMENTED** [review]
Thanks for tidying up the DG!

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -341,43 +322,31 @@ Given below is an example success scenario and how the `export` mechanism behave
...
+The following sequence diagram shows how the `import` command works:
 
 ![ExportSequenceDiagram](images/ExportSequenceDiagram.png)
    
 #### Design Considerations
 
-**Aspect: Where to save the exported JSON file**
```
Comment (maryjess) [inline docs/DeveloperGuide.md:350]: Just realised the Design Consideration for Export feature is now missing

**Review by maryjess — CHANGES_REQUESTED** [review]

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -713,29 +716,140 @@ testers are expected to do more *exploratory* testing.
...
-   1. Prerequisites: List all persons using the `list` command. Multiple persons in the list.
+### 7.5 Deleting an applicant
+
+1. Deleting an applicant while all applicants are being shown
+
+   1. Prerequisites: List all applicants using the `list` command. Multiple applicants in the list.
 
    2. Test case: `delete 1`<br>
-      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
+      Expected: First contact is applicant from the list. Details of the deleted applicant shown in the status message.
```
Comment (cliftonfelix) [inline docs/DeveloperGuide.md:772]: Please fix the mistake here

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -772,7 +772,7 @@ testers are expected to do more *exploratory* testing.
       Expected: First applicant is deleted from the list. Details of the deleted applicant shown in the status message.
       Success message shown in the status message.
 
-2. Deleting an out-of-bounds index
+2. Deleting an non-positive index
```
Comment (cliftonfelix) [inline docs/DeveloperGuide.md:775]: Should be a

**Review by cliftonfelix — APPROVED** [review]
LGTM

**Pull request merged**